examples/helpers/example.js.tmpl

```diff
@@ -22,9 +22,7 @@ const ethPrivKey = envVars.ETH_PRIVATE_KEY
 const starkPrivKey = envVars.STARK_PRIVATE_KEY
 const rpcUrl = envVars.RPC_URL
 
-const provider = new HDWalletProvider(ethPrivKey, rpcUrl)
-const web3 = new Web3(provider)
-provider.engine.stop()
+const { web3, provider } = getWeb3(ethPrivKey, rpcUrl)
```

Comment:
provider doesn't seem to be used anywhere 
```suggestion
const { web3 } = getWeb3(ethPrivKey, rpcUrl)
```

Reply:
indeed, I only exported it in case there was a usecase to prevent a mass change in the future